src/components/PageBlocks/CalendarBase.vue

```diff
@@ -301,12 +303,14 @@ export default {
 
                 return compose.PageBlockCalendar.RecordFeed(this.$ComposeAPI, module, this.namespace, ff, this.loaded)
                   .then(events => {
+                    this.events.length = 0
```

Comment:
Does settings the length property of an array make sense?

Reply:
It is one of the best practice way to empty an array.
![image](https://user-images.githubusercontent.com/31524469/200049783-1bd1d691-fbbc-473d-a62d-df8a4a265fa3.png)